src/test/java/seedu/duke/model/ItemTest.java

```diff
@@ -0,0 +1,97 @@
+package seedu.duke.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import seedu.duke.model.exception.InvalidFormat;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+class ItemTest {
+    Item testItem;
+
+    @BeforeEach
+    void prepare() {
+        testItem = new Item("Dune", "1.00", "2.00");
+    }
+
+    @Test
+    void setName() {
```

Comment:
Edit the following test names to follow the test method format such as methodBeingTested_inputGiven_expectedOutcome()

Reply:
Thanks i have changed the test methods accordingly